modules/fb_instant_articles_display/src/EntityPropertyMapper.php

```diff
@@ -109,16 +109,27 @@ private function addHeaderFromProperties() {
       );
     }
 
+    // user token or static string for author name
```

Comment:
Capitalize comment and end with punctuation. See Drupal's [API documentation and comment standards](https://www.drupal.org/coding-standards/docs#drupal).